### PR TITLE
Fixing online counts. Fixes #664

### LINKED ISSUE #664: Fix online users count.
This got broken as I've moved to injecting the websocket.

Hide it until then.

### PR DESCRIPTION
**Pull request by dessalines**

### CODE REVIEW

File: server/src/api/community.rs [hunk 1]
```diff
@@ -203,13 +203,10 @@ impl Perform for Oper<GetCommunity> {
...
-      // TODO
-      1
-    // let fut = async {
-    //   ws.chatserver.send(GetCommunityUsersOnline {community_id}).await.unwrap()
-    // };
-    // Runtime::new().unwrap().block_on(fut)
+      ws.chatserver
+        .send(GetCommunityUsersOnline { community_id })
+        .await
+        .unwrap_or(1)
```
Comment (Nutomic) [inline server/src/api/community.rs:209]: Is there a particular reason why you are returning 1 instead of logging an error?

**Comment by Nutomic** [comment]
Also I'm wondering how this fixes the problem, because the code looks exactly identical to whats commented out.

File: server/src/api/community.rs (hunk 1 above)
Comment (dessalines) [reply server/src/api/community.rs:209]: Ya, I don't want the whole API method to fail if websocket stuff doesn't get sent / received for whatever reason... its not vital.

**Comment by dessalines** [comment]
I think I wrote it before / in the middle of @asonix 's async additions, and couldn't use await.

**Comment by asonix** [comment]
If you don't care about results from actors, use do_send instead of send

**Comment by dessalines** [comment]
I do care about the results for this one, to get the counts.

**Pull request merged**